src/main/java/at/hannibal2/skyhanni/config/features/foraging/ForagingTrackerConfig.kt

```diff
@@ -29,6 +31,38 @@ class ForagingTrackerConfig {
     @FeatureToggle
     var compactGiftChats: Boolean = true
 
+
+    @Expose
+    @ConfigOption(
+        name = "Compact Gifts Bonus Drops",
+        desc = "Lets you decide what types of bonus drops should be included in Compact Gifts messages."
+    )
+    @ConfigEditorDraggableList
+    val compactGiftBonusDropsList: Property<MutableList<TreeGiftBonusDropCategory>> = Property.of(
```

Comment:
Do we NEED to have this list be a property?
Do you intend to add logic to update the display when the list gets changed? Currently, I don't see this logic in your PR.
So we could right now remove the property wrapper. The final version of this here would look like this then:
https://github.com/hannibal002/SkyHanni/blob/8178226e8ee631ebc10cda179ee9c2de13303d19/src/main/java/at/hannibal2/skyhanni/config/features/chat/PlayerMessagesConfig.kt#L22